activation_api.py

```
@@ -252,3 +252,54 @@ async def activation_audit_log(request: Request, limit: int = 100) -> list[dict]
+
+@activation_router.post("/users/{user_id}/role")
+async def change_user_role(
+    user_id: str,
+    body: _RoleUpdateBody,
+    request: Request,
+) -> dict:
+    """Change the role of a registered user (admin only).
+
+    Allowed roles: ``user``, ``power_user``, ``admin``.
+    """
+    admin_id = _require_admin(request)
```

Ruff (0.15.13)

[error] Undefined name `_require_admin`

(F821)